src/Intacct/ClientConfig.php

```diff
@@ -254,6 +254,40 @@ public function setUserPassword(string $userPassword)
         $this->userPassword = $userPassword;
     }
 
+    private $headers;
+
+    /**
+     * @return null|array
+     */
+    public function getHeaders() : ?array
+    {
+        return $this->headers;
+    }
+
+    /**
+     * @param array $headers
+     */
+    public function setHeaders(array $headers) : void
+    {
+        $this->headers = $headers;
+    }
+
+    /**
+     * @param string $key
+     * @param string $value
+     */
```

Comment:
```suggestion
    public function getHeaders() : ?array
    {
        return $this->headers;
    }

    public function setHeaders(array $headers) : void
    {
        $this->headers = $headers;
    }

```

Don't need docblocks!